servers/src/main/java/tachyon/worker/block/meta/StorageDir.java

```diff
@@ -124,7 +124,7 @@ private void initializeMeta() throws BlockAlreadyExistsException, IOException,
           // TODO(calvin): Resolve this conflict in class names.
           org.apache.commons.io.FileUtils.deleteDirectory(path);
         } catch (IOException ioe) {
-          LOG.error("can not delete directory {}: {}", path.getAbsolutePath(), ioe);
+          LOG.error(String.format("can not delete directory %s", path.getAbsolutePath()), ioe);
```

Comment:
for a single param log, you can achieve the same result in a cleaner way using 
`LOG.error("can not delete directory {}", path.getAbsolutePath(), ioe);`
which avoids using `String.format`.
This is supported after slf4j 1.7 
http://www.slf4j.org/faq.html#paramException